### ISSUE TITLE
external pipeline command launched twice

### ISSUE THREAD

**Issue by vvo** [middleman/middleman#1858]
When using

```rb
  activate :external_pipeline,
    name: :webpack,
    command: "webpack build into .webpack/javascripts/file.js",
    source: ".webpack"
```

No matter if I use livereload or not, the command will be launched twice. Is this expected behavior?

Thanks.

**Comment by tdreyno** [comment]
Should have been fixed in v4.1.4. Are you up to date?

**Comment by vvo** [comment]
Yes it did, thanks
